plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-typography.tsx

```diff
@@ -201,18 +202,16 @@ export const SidebarNavigationScreenTypography = () => {
 										</Button>
 										<Button
 											onClick={ async () => {
```

Comment:
Is the `async` still needed here or can this be a regular arrow fn now that we removed the `await` portion from the function body?

Reply:
Nice catch! Fixed!